server/jetstream.go

```diff
@@ -331,34 +351,35 @@ func (a *Account) EnableJetStream(limits *JetStreamAccountLimits) error {
 			metafile := path.Join(tdir, fi.Name(), JetStreamMetaFile)
 			metasum := path.Join(tdir, fi.Name(), JetStreamMetaFileSum)
 			buf, err := ioutil.ReadFile(metafile)
+			errEvent := jsStartUpError.Detail(JSErrorDetail{"metafile": metafile, "metasum": metasum})
 			if err != nil {
-				s.Warnf("  Error reading StreamTemplate metafile %q: %v", metasum, err)
+				a.sendJSError(s, errEvent.UserMessage("Error reading StreamTemplate metafile").Wrap(err))
 				continue
 			}
 			if _, err := os.Stat(metasum); os.IsNotExist(err) {
-				s.Warnf("  Missing StreamTemplate checksum for %q", metasum)
+				a.sendJSError(s, errEvent.UserMessage("Missing StreamTemplate checksum"))
```

Comment:
Is `.Wrap()` or `.Detail()` missing here or not?

Reply:
No I dont think there's much value in wrapping the error from stat in this case, so just a basic error, we already know its missing and thats what the err is.